### PR TITLE
Feature/masthead

### PR DESCRIPTION
**Pull request by Fodilahy-mena**
Dear Shaun,

We've added a new pattern!

**Description**

We are very happy to finish our task in masthead. We hope it is what it meant to be and please have a glimpse at our PR, feeling fantastic to have a change if needed.
 
We have faced many problems that are mostly solved:

- We struggled with the image element in html but we solved it by wrapping the image inside the <picture>element.

- Other problem about the image was the images that we download were so heavy. We pilled it up by using decode command in git bash.

- Checking our code with some browsers that we have on our computer and the Michrosoft Edge doesn't support the logical properties so we had to use the shorthand of padding.
  

- In addition, we had problem about the background so we had to use longhand background-color with the percentage from the origin in the figma.

**Screenshots**

image

**Checklist**

- [x] Merged to `testing`
- [x] Cross-browser tested
- [x] Files are in the right places
- [x] Suitable names have been chosen for `class`es, `id`s and files.
- [x] New patterns have a corresponding HTML file, that has all the relevant test cases within.

We are so thankful by having this exercise because we learned many things with both css and html.

Warm regard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 We'll be working in feature branches.
 
-Create a new branch called: `feature/PATTERN_NAME`.
+Create a new branch called: `feature/masthead`.
```
Comment (ndorfin) [inline README.md]: Ooops, this should be generic. Please undo this.

File: css/masthead.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+/*header to hold the property needed for all contents*/
+.masthead > header {
+  font-style: normal;
+  text-align: center;
+  padding-top: 4rem;
+  padding-bottom: 4rem;
+  padding-left: 4rem;
+  padding-right: 4rem;
+  background-color: rgba(0,131,195, 80%);
```
Comment (ndorfin) [inline css/masthead.css:14]: Problems with the `a` value of this `rgba` function.

File: css/masthead.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+/*we put position absolite here in order to make the image to set behind the headings*/
+picture.masthead__image{
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+}
+
```
Comment (Mickaellah) [inline css/masthead.css:45]: You should not use any element as a selector while there is a class that you can use.

File: css/masthead.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+body > main > .masthead__donation > header , .masthead__hire > header {
+  background-color: rgba(124, 120, 120, 0.466);
+}
+
+body > main > .masthead__donation > header > .masthead__donation--heading,
+body > main > .masthead__donation > header > .masthead__donation--description,
+body > main > .masthead__hire > header > .masthead__hire--heading,
+body > main > .masthead__hire > header >.masthead__hire--description {
+  text-shadow: 2px 2px 3px black;
+  }
```
Comment (Mickaellah) [inline css/masthead.css:66]: Do not use long selectors if you have classes.

File: css/masthead.css [hunk 5]
```diff
@@ -5,11 +5,32 @@ body > main > .masthead {
...
+  color: #FFFFFF;
+}
+
+p {
+  font-family: Maison Neue Mono Trial;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 30px;
+  letter-spacing: 0.45px;
+  color: #FFFFFF;
```
Comment (Mickaellah) [inline css/masthead.css]: This selector is too broad. This may break everyone else's codes. Please make a change.

File: css/masthead.css [hunk 6]
```diff
@@ -41,6 +44,8 @@ picture.masthead__image{
   bottom: 0;
   z-index: -1;
 }
+
+/*we specified the img element to cover the whole screen*/
 img {
   width: 100%;
   height: 100%;
```
Comment (Mickaellah) [inline css/masthead.css:49]: It's nice that you added comments in your css.

File: masthead.html [hunk 7]
```diff
@@ -25,6 +25,9 @@ <h1>Onja</h1>
           <h2 class="masthead__heading">About us</h2>
 
           <p class="masthead__description">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus enim corporis laboriosam placeat possimus, voluptatum culpa dignissimos laudantium ipsam voluptatibus quibusdam tempore asperiores beatae non neque omnis rerum fugit aperiam.
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nesciunt iusto blanditiis ad animi, laborum, dicta rem, tempore libero ullam fugiat necessitatibus laudantium? Exercitationem veniam quasi fugiat aliquid iure aspernatur ullam.
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Enim, vitae dicta iste velit quaerat, ratione doloremque, tenetur esse minus nesciunt hic in aut modi aliquam atque laudantium quo? A, consectetur!
             Proper cookery renders good food material more digestible. When scientifically done,
           </p>
         </header>
```
Comment (Mickaellah) [inline masthead.html]: You have got it made.

**Comment by Mickaellah** [comment]
Great job team! But there are few things which need to be changed. Please make sure your spellings (faulder, converd, backfround). If you have classes, do not use the tag names, use your classes.

**Review by Mickaellah — COMMENTED** [review]
Do not forget to use the class name.

**Pull request closed without merging**